webapp/pages/profile/_id/_slug.vue

```diff
@@ -244,8 +244,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-disabled="$apollo.loading"
+          :infinite-scroll-distance="10"
+          :infinite-scroll-throttle-delay="800"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-26T09:39:56Z" title="Monday, August 26th 2019, 11:39:56 am +02:00">Aug 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
+          :infinite-scroll-disabled="$apollo.loading"
+          :infinite-scroll-distance="10"
+          :infinite-scroll-throttle-delay="800"
```


default is 200ms... what went into the thinking to make it `800`? just your feeling? or was there something specific?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-26T10:10:08Z" title="Monday, August 26th 2019, 12:10:08 pm +02:00">Aug 26, 2019</time>_

---


It was try/error with @alina-beck. We tested it with seed data. Maybe we can increase it even further after we learned about the low performance of our neo4j pod.